content/en/cloudcraft/getting-started/connect-amazon-eks-cluster-with-cloudcraft.md

```diff
@@ -18,7 +18,9 @@ To connect your AWS account and familiarize yourself with Cloudcraft, see the fo
 - [Connect your AWS account with Cloudcraft][4]
 - [Create your first live AWS diagram][5]
 
-You should also [install and configure `kubectl`][7], a tool that allows you to control Kubernetes clusters through the command line. Cloudcraft recommends using the latest version to avoid issues.
+[Install and configure `kubectl`][7], a tool that allows you to control Kubernetes clusters through the command line. Cloudcraft recommends using the latest version to avoid issues.
+
+In addition, Cloudcraft cannot scan private clusters or clusters with IP filtering. Scanning won't work if the public access source allow list option in the EKS networking configuration is modified from the default of `0.0.0.0/0`.
```

Comment:
Can we convert this into **pre-requisites** and phrase it as what we can need 
For example:
In order to scan your EKS cluster succesfully, Cloudcraft requires clusters to have public access enabled and no IP filtering applied. The Public Access Source Allow List option in the EKS networking configuration must remain set to its default value of 0.0.0.0/0.

Reply:
Fixed with [36c5987](https://github.com/DataDog/documentation/pull/26555/commits/36c5987d703a6b9449da8d2e4c4c5f0579fa6dee).